environments/content02/inventory

```diff
@@ -54,32 +54,36 @@ content02
 [press:children]
 content02
 
+[press_worker:children]
+content02
+
 # Groups of groups
 
 [broker_connected:children]
 publishing
 channel_processing
 publishing_worker
+press
+press_worker
 
 [db_connected:children]
 archive
 publishing
 channel_processing
 publishing_worker
-authoring
 zclient
 pdf_gen
 backup
 press
+press_worker
 
 [nfs_connected:children]
-archive
```

Comment:
This is not: we still need access to the NFS mounted file exports - archive uses FS access to list what downloadable files are available